tests/Role/ReadModel/Permissions/AppConfigUserPermissionsReadRepositoryTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Role\ReadModel\Permissions;
+
+use CultuurNet\UDB3\Role\ReadModel\Permissions\Doctrine\UserPermissionsReadRepository;
+use CultuurNet\UDB3\Role\ValueObjects\Permission;
+use PHPUnit\Framework\TestCase;
+
+final class AppConfigUserPermissionsReadRepositoryTest extends TestCase
+{
+    private AppConfigUserPermissionsReadRepository $repository;
+
+    protected function setUp(): void
+    {
+        $userPermissionsReadRepository = $this->createMock(UserPermissionsReadRepository::class);
+        $config = [
+            'jkfhsjkfsdhjk@clients' => [Permission::aanbodBewerken(), Permission::productiesAanmaken()],
+        ];
+
+        $this->repository = new AppConfigUserPermissionsReadRepository($userPermissionsReadRepository, $config);
+    }
+
+    /**
+     * @test
+     */
+    public function it_returns_all_enabled_permissions_for_client_id_when_executing_get_permissions()
```

Comment:
I would still add the return type, even if it is `void` (same for the rest of the file).